lambdas/answer/build.gradle

```diff
@@ -5,13 +5,12 @@ plugins {
 
 dependencies {
 	implementation project(":lib")
-	implementation configurations.lambda
-	implementation configurations.jackson
-	implementation configurations.dynamodb
-	implementation configurations.log
-
-	implementation configurations.powertools
-	testImplementation configurations.tests
+	implementation configurations.lambda,
+			configurations.jackson,
+			configurations.dynamodb,
+			configurations.powertools
```

Comment:
Have a look at address_cri, it something like `aspect configurations.powertools`